homeassistant/components/template/vacuum.py

```diff
@@ -148,7 +148,9 @@ def __init__(
         self._template = config.get(CONF_VALUE_TEMPLATE)
         self._battery_level_template = config.get(CONF_BATTERY_LEVEL_TEMPLATE)
         self._fan_speed_template = config.get(CONF_FAN_SPEED_TEMPLATE)
-        self._attr_supported_features = VacuumEntityFeature.START
+        self._attr_supported_features = (
+            VacuumEntityFeature.START | VacuumEntityFeature.STATE
```

Comment:
why not setting it directly for all in the class StateVacuumEntity?

Reply:
Setting it in the base class is not straight forward, we should remove it instead.

There's some background and a suggested way forward here: https://github.com/home-assistant/core/pull/95832#issuecomment-1620054106